tests/scripts/thread-cert/node.py

```diff
@@ -1751,7 +1751,7 @@ def scan(self, result=1):
             return self._expect_results(
                 r'\|\s(\S+)\s+\|\s(\S+)\s+\|\s([0-9a-fA-F]{4})\s\|\s([0-9a-fA-F]{16})\s\|\s(\d+)')
 
-    def ping(self, ipaddr, num_responses=1, size=None, timeout=5):
+    def ping(self, ipaddr, num_responses=1, size=None, timeout=10):
```

Comment:
why do we change timeout?

Reply:
Our default ping command timeout is 5 seconds. Using the same timeout here may not be sufficient for the test to wait for ping command to complete.

Reply:
Fine to me as long as TC passes.